lighthouse-core/test/scripts/i18n/collect-strings-test.js

```diff
@@ -136,17 +163,21 @@ describe('Compute Description', () => {
        * @example {Variable example.} variable
        * @example {Variable2 example.} variable2
        */
-      message: 'Hello World {variable} {variable2}',
+      messageStr: 'Hello World {variable} {variable2}',
     };`;
+    const liveUIStrings = evalJustUIStrings(justUIStrings);
+    const res = collect.parseUIStrings(justUIStrings, liveUIStrings);
 
-    const ast = esprima.parse(justUIStrings, {comment: true, range: true});
-
-    const stmt = ast.body[0];
-    const prop = stmt.declarations[0].init.properties[0];
-    const res = collect.computeDescription(ast, prop, 'Hello World {variable} {variable2}', 0);
-    expect(res.description).toBe('Tagged description for Hello World.');
-    expect(res.examples['variable']).toBe('Variable example.');
-    expect(res.examples['variable2']).toBe('Variable2 example.');
+    expect(res).toEqual({
+      messageStr: {
```

Comment:
is there significance to all of these being called `messageStr`?

Reply:
> is there significance to all of these being called `messageStr`?

no, I just kept confusing myself with `message` as a property on the compared object *and* as a property on the `message` property. If this renames it it should probably rename it something totally different instead of being super lazy like this